src/main/java/net/minecraftforge/common/config/ConfigElement.java

```diff
@@ -360,4 +360,40 @@ public Object getMaxValue()
     {
         return isProperty ? prop.getMaxValue() : null;
     }
+    
+    public static ConfigElement from(String modid)
+    {
+        return from(modid, null);
```

Comment:
The idea was, that in the @Config annotation, you don't have to specify a name if you want to work with the default config file. But as I am going to change that (I didn't think about just passing the Class with the annotation) this will be obsolete.